content/docs/tutorials/cloud-tutorial.md

```diff
@@ -0,0 +1,562 @@
+* `--raw`					– specifies that the incoming data should be stored as a single record.
+* `--key-separator <key>`	– specifies the string delineator to split the input into a key and value pairing.
+
+#### Consume
+
+`fluvio consume` is the main way to read data from the Fluvio database. Either
+in scripting, or through the use of the CLI, most interactions will use `consume` in some way.
+
+Here is an example of `consume` in action:
+
+%copy first-line%
+```bash
+$ fluvio consume greetings -dB
+Consuming records from the beginning of topic 'greetings'
+Hello world!
+test
+meow
+```
+
+-> To quit `consume` press `CTRL-C`.
+
+`fluvio consume <topic> [flags]` by default prints new records to the terminal as
+they enter the database. By default it runs nonstop until quit – the examples
+used here all use the `-d` flag to tell it to stop.
+
+Some useful option flags to be aware of:
+
+* `d`						– halts consumption after reaching the end of the records available.
+* `T[int]`					– consumes only the T (default 10) most recent records.
+* `B[int]`					– starts consuming records B (default 0) after the start of the database.
+* `-p[int]`					– reads only from the partition p (default 0).
+* `-k`						– displays the key portion of the key and value pairs.
+* `-A`						– reads from all partitions available.
+* `--smart-module <module>` – runs a [SmartModule](#smartmodules) module on the output, then displays the results.
+
+## An Easy Fluvio Script — Bash
+
+Here is a simple script that pushes timestamped comments to a remote database.
+
+This code generates a message string that contains the current time, and the
+contents of the first argument passed to it.
+
+Once done, it checks to see if the hard coded topic exists. If it
+does, good, it moves on. If not, it calls upon `fluvio topic create`
+to generate the new topic. Then, it makes a call to `fluvio produce`
+and sends the timestamped message off to the Cloud.
+
+{{<code file="code/bash/timekeeper.sh" lang="bash" copy=true >}}
+
+##### To Run the Script
+
+Save the above script to a file – here it is named `timekeeper.sh` – and set it
+as executable.
+
+You can then run the script with:
+
+%copy first-line%
+```bash
+$ ./timekeeper.sh 'I love cupcakes'
+topic "timekeeper" created
+```
+
+Again, you can use `consume` to view the results.
+
+%copy first-line%
+```bash
+$ fluvio consume timekeeper -dB
+Consuming records from the beginning of topic 'timekeeper'
+2022-09-01T18:46:12-07:00 : I love cupcakes
+```
+
+## Connectors
+
+If you wish to automatically collect information from one source and send it to
+Fluvio, Connectors are the way to go. When given the information on the
+interface through the Connector configuration file, Fluvio can poll a multitude of
+input types.
+
+In this tutorial, we will be looking at the [HTTP Connector](/connectors/sources/http/) setup, connecting
+to the `catfacts.ninja` JSON database.
```

Comment:
We should link to the site, but it might need to be a raw html link, because I want it to open in a new window, and make it a `Nofollow` link, so we're not strengthening our search ranking to cat facts.

We should do this with all of the catfact.ninja links.

ex. `<a href="https://catfact.ninja" target="_blank" rel="nofollow>`